src/harness/testcases/grant_testcase.py

```diff
@@ -253,6 +253,56 @@ def test_WINNF_FT_S_GRA_12(self):
     self.assertTrue(response['response']['responseCode'] in (103, 105))
 
   @winnforum_testcase
+  def test_WINNF_FT_S_GRA_13(self):
+    """Send grant request for device with cbsdId different from its assigned cbsdId,
+        SAS should reject grant.
+    The responseCode should be (103, 105), with NO channels in result.
+    """
+    device_a_cert = os.path.join('certs', 'device_a.cert')
+    device_a_key = os.path.join('certs', 'device_a.key')
+
+    device_c_cert = os.path.join('certs', 'device_c.cert')
+    device_c_key = os.path.join('certs', 'device_c.key')
+
+    # Checks that devices certificate are valid (checking all chain).
+    self.assertValidClientCertificate(device_a_cert, sas.CA_CERT)
```

Comment:
I think we can skip this check, as it's not asked for in the test spec. That will also let you revert the changes which redefine assertValidClientCertificate().